services/consensuscontext/committee.go

```diff
@@ -42,3 +42,30 @@ func (s *service) RequestValidationCommittee(ctx context.Context, input *service
 	}
 	return res, nil
 }
+
+
+func (s *service) RequestBlockProofOrderingCommittee(ctx context.Context, input *services.RequestBlockProofCommitteeInput) (*services.RequestBlockProofCommitteeOutput, error) {
```

Comment:
1) i would add here a comment telling that this is different because it doesn't care about order just who is. (committee)
2) are we 100% sure that we always ask a committee available by ref ? management is only "per-day" ?